### PR TITLE
Update the readme

### PR DESCRIPTION
**Pull request by rwestberg**
It never hurts to update the readme.

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Download
`$ git fetch https://git.openjdk.java.net/playground pull/1/head:pull/1`
`$ git checkout pull/1`

### CODE REVIEW

**Comment by rwestberg-test** [comment]
Test

File: README [hunk 1]
```diff
@@ -2,6 +2,10 @@
 Welcome to the JDK!
 ===================
 
+Note that this is a playground repository, and may be synced and/or
```
Comment (rwestberg-test) [inline README:5]: Test 2

File: README [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+reset at any time without notice. But feel free to experiment with
+creating pull requests and integrating them!
```
Comment (rwestberg-test) [inline README:7]: Test 3

File: README (hunk 1 above)
Comment (rwestberg-test) [inline README:5]: Test 4

**Review by rwestberg-test — COMMENTED** [review]
Test 5

**Review by lgxbslgx — APPROVED** [review]

**Review by lgxbslgx — APPROVED** [review]

**Pull request closed without merging**